lib/process-services/src/lib/form/widgets/dropdown/dropdown.widget.spec.ts

```diff
@@ -126,6 +126,80 @@ describe('DropdownWidgetComponent', () => {
         expect(widget.field.options[1]).toBe(restFieldValue);
     });
 
+    it('should not display required error for a non required dropdown when selecting the none option', async () => {
```

Comment:
do we have a test for the opposite case? make sure we have an error when is required?